aws/resource_aws_ssm_maintenance_window_target_test.go

```diff
@@ -160,6 +190,8 @@ resource "aws_ssm_maintenance_window" "foo" {
 }
 
 resource "aws_ssm_maintenance_window_target" "target" {
+	name = "TestMaintenanceWindowTarget"
```

Comment:
Looks like we have some extra white space here, and in some of the other test configs.